Out in the open september 2023 (#482)
okfde/codefor.de: content/blog/2023-10-03-out-in-the-open-september-2023.md

Schon [im Juli](https://codefor.de/blog/out-in-the-open-july-2023/) berichteten wir vom schleppenden Vorankommen beim Bundestransparenzgesetz. Terminliche Versprechungen wurden nicht eingehalten; das Gesetz scheint für die Koalition in die zweite Reihe gerückt zu sein. Nun fordert ein [Bündnis aus Nichtregierungsorganisationen in einer gemeinsamen Pressemitteilung](https://netzpolitik.org/2023/transparenzgesetz-wird-das-noch-was-frau-faeser/) Fortschritte.

Kurz vor den Landtagswahlen in Hessen und Bayern mag es für einige Wähler:innen durchaus von Interesse sein, wie die einzelnen Parteien zum Thema Transparenz stehen. Auf [FragDenStaat](https://fragdenstaat.de/blog/2023/09/26/in-hessen-und-bayern-wird-gewahlt-so-stehen-die-parteien-zur-transparenz/) lassen sich die parteilichen Position [anhand sog. Wahlprüfsteine](https://fragdenstaat.de/aktionen/wahlpruefsteine/) nachvollziehen.

> Passend dazu: Das GitHub-Repository [qual-o-mat-data](https://github.com/gockelhahn/qual-o-mat-data) stellt Daten des [Wahl-O-Mats](https://www.wahl-o-mat.de/) der [Bundeszentrale für politische Bildung (BPB)](https://www.bpb.de/themen/wahl-o-mat/) zur Verfügung. Zwar veröffentlicht die BPB die Daten seit zwei Jahren im Excel-Format. Allerdings sind die Datensammlungen nicht strukturiert und Daten von vor 2021 sind erst gar nicht enthalten. Im Repository lassen sich alle Daten seit 2002 im JSON-Format beziehen, geordnet nach Jahren und Bundesländern. Wer eine Gesamtübersicht im Textformat bevorzugt, wird [an anderer Stelle](https://pastes.io/1fhj4spoat) ebenfalls fündig.

Reviewer: ```suggestion
Passend dazu: Das GitHub-Repository [qual-o-mat-data](https://github.com/gockelhahn/qual-o-mat-data) stellt Daten des [Wahl-O-Mats](https://www.wahl-o-mat.de/) der [Bundeszentrale für politische Bildung (BPB)](https://www.bpb.de/themen/wahl-o-mat/) zur Verfügung. Die BPB veröffentlicht die Daten seit zwei Jahren im Excel-Format, allerdings sind die Datensammlungen nicht strukturiert und Daten vor 2021 erst gar nicht enthalten. Im Repository lassen sich alle Daten seit 2002 in einem JSON-Format beziehen, geordnet nach Jahren und Bundesländern. Wer eine Gesamtübersicht im Textformat bevorzugt, wird [an anderer Stelle](https://pastes.io/1fhj4spoat) ebenfalls fündig.
```

Die Klimakrise bringt klimatische Veränderungen mit sich, die mittlerweile auch in Mitteleuropa zu spüren sind. Wer wissen möchte, wie sich die Verschiebung der klimatischen Bedingungen weiterentwickelt, kann sich eine [Karte von Johannes Christ](https://observablehq.com/embed/f487ad0bc3086501@209?cells=viewof+stadt_select%2Cviewof+zeit_select%2Cklimaanalogien_map%2Cstyle) anschauen. Sie blickt sowohl in die Vergangenheit als auch in die Zukunft und basiert auf Daten des Umweltbundesamtes.

Gerade im Mobilitätssektor schlummern große Potenziale zum Einsparen von Emissionen. Dazu beitragen könnte die Stärkung des Radverkehrs - wenn denn die Fortbewegung mit dem Drahtesel attraktiv ist. Die Realität jedoch besteht vielerorts aus schlecht ausgebauten Wegen und hohen Unfallzahlen: In der Schweiz sticht insbesondere Zürich heraus, wenn es um die Anzahl der Fahrradunglücke geht. Ein [Team um den Journalisten Simon Huwiler](https://twitter.com/simon_huwiler/status/1701560427321143477?s=20) hat deshalb auf Basis offener Daten [eine interaktive Unfallkarte veröffentlicht](https://www.nzz.ch/zuerich/so-gefaehrlich-ist-ihre-zuercher-veloroute-ld.1749405).

> Immerhin bemühen sich immer mehr Städte, den Bedarf nach Radwegen sichtbar zu machen. In zahlreichen Orten wachsen Raddauerzählstellen aus dem Boden, welche die Anzahl der Radelnden messen. Nicht immer sind die gesammelten Daten frei verfügbar - in München werden die Messungen der Zählstellen nun aber [auf dem stadteigenen Open Data-Portal zur Verfügung gestellt](https://opendata.muenchen.de/dataset/daten-der-raddauerzaehlstellen-muenchen-2023).

Reviewer: Die Daten gibts schon relativ lang, mindestens ab [2017](https://opendata.muenchen.de/dataset/daten-der-raddauerzaehlstellen-muenchen-maerz-2017).
```suggestion
Immerhin bemühen sich immer mehr Städte, den Bedarf nach Radwegen sichtbar zu machen: In zahlreichen Orten wachsen Raddauerzählstellen aus dem Boden, welche die Anzahl der Radelnden messen. Nicht immer sind die gesammelten Daten frei verfügbar - in München werden die Messungen der Zählstellen [auf dem stadteigenen Open Data-Portal zur Verfügung gestellt](https://opendata.muenchen.de/dataset/daten-der-raddauerzaehlstellen-muenchen-2023).
```

Überhaupt beschäftigen sich immer mehr Open Data-Projekte mit dem Thema Radverkehr. So zum Beispiel die [Karte der infraVelo aus Berlin](https://www.infravelo.de/karte/): Neben dem Radwegenetz kann man hier Projekte ausfindig machen, die der Verbesserung der Infrastrutur dienen.

> Dass sich der Klimaschutz nicht einzig und allein auf die Stärkung des Radverkehrs konzentrieren sollte, ist mittlerweile allen klar - doch in welchen Bereichen bringt die Einsparung von CO2 am meisten? Ein Team des BR hat den öffentlich zugänglichen [Projektionsbericht des Umweltbundesamtes](https://www.umweltbundesamt.de/themen/klima-energie/klimaschutz-energiepolitik-in-deutschland/szenarien-fuer-die-klimaschutz-energiepolitik/integrierte-energie-treibhausgasprojektionen#undefined) analysiert und daraus [einen CO2-Rechner](https://interaktiv.br.de/co2-rechner/) abgeleitet. Er zeigt, [welche Maßnahmen wirklich wie viele Emissionen einsparen können](https://www.br.de/nachrichten/deutschland-welt/datenanalyse-was-beim-co-sparen-am-meisten-bringt,Tpq6q6O).

Reviewer: ```suggestion
Dass sich der Klimaschutz nicht einzig und allein auf die Stärkung des Radverkehrs konzentrieren sollte, ist mittlerweile allen klar - doch in welchen Bereichen bringt die Einsparung von CO2 am meisten? [BR Data](https://www.br.de/extra/br-data/) hat den öffentlich zugänglichen [Projektionsbericht des Umweltbundesamtes](https://www.umweltbundesamt.de/themen/klima-energie/klimaschutz-energiepolitik-in-deutschland/szenarien-fuer-die-klimaschutz-energiepolitik/integrierte-energie-treibhausgasprojektionen#undefined) analysiert und daraus [einen CO2-Rechner](https://interaktiv.br.de/co2-rechner/) abgeleitet. Er zeigt, [welche Maßnahmen wirklich wie viele Emissionen einsparen können](https://www.br.de/nachrichten/deutschland-welt/datenanalyse-was-beim-co-sparen-am-meisten-bringt,Tpq6q6O).
```
